guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
```

Comment:
It can be a separate cost e.g. recommending nearby canteens/restaurants that can accommodate participants. Same idea with lodging recommendations. But IMO, that'll be harder to plan out and will give participants a harder time to get food (especially international guests). For me, it's better to get an average prepared lunch than to scurry around looking for food.